utils/resource/usage.go

```diff
@@ -205,6 +249,12 @@ type proc struct {
 	// [lastWriteBytes] is the most recent measurement of total disk bytes
 	// written.
 	lastWriteBytes uint64
+
+	// number of CPUCycles
+	cpuCycles float64
```

Comment:
`lastTotalCPU` is already the number of CPUCycles - no?

Reply:
done